Add theme documentation (#5)
mautic/developer-documentation-new: docs/themes/system.rst

> Overriding Core View Templates

Reviewer: ```suggestion
Overriding core view templates
```

==========================================================

> One can override any view's template in Mautic by creating a ``themes/system`` directory, copy the core's template into a corresponding directory and customizing the HTML.

Reviewer: ```suggestion
You can override any view's template in Mautic by creating a ``themes/system`` directory, copying the core template file into a corresponding directory and customizing the HTML.
```

Let's be consistent with tenses.

> For example, to customize the login page. Copy ``app/bundles/UserBundle/Views/Security/login.html.php`` into ``themes/system/MauticUserBundle/Security/login.html.php`` then make the desired changes.

Reviewer: ```suggestion
For example, to customize the login page, copy ``app/bundles/UserBundle/Views/Security/login.html.php`` into ``themes/system/MauticUserBundle/Security/login.html.php`` then make the desired changes.
```

> .. Warning:: Future Mautic upgrades may break if core templates were updated. Overridden core templates must be reconciled with changes done in future upgrades.

Reviewer: ```suggestion
.. Warning:: Future Mautic upgrades may break if core templates were updated. Overridden core templates must be reconciled with changes in future upgrades.
```
